### PR TITLE
https://issues.redhat.com/browse/ACM-11938--eus to eus feature broken…

### PR DESCRIPTION
**Pull request by swopebe**
… in to steps and the What's new entry

### CODE REVIEW

File: clusters/cluster_lifecycle/ansible_config_cluster.adoc [hunk 1]
```diff
@@ -178,3 +177,64 @@ metadata:
...
+spec:
+  desiredCuration: upgrade
+  upgrade:
+    intermediateUpdate: 4.13.x
+    desiredUpdate: 4.14.x
+    monitorTimeout: 120
+----
+
+. Enter `Unpause machinepool` in the posthook job, and `pause machinepool` in the `prehook` job:
+//we should say why we want to do this.
```
Comment (swopebe) [inline clusters/cluster_lifecycle/ansible_config_cluster.adoc]: Please look at line 200 and provide a little more info?

**Comment by swopebe** [comment]
@fxiang1 this is ready for your review. Please notice that I took what was important from the example you gave me and the summary, and looked at the feature itself to break this into a task with examples. I also took your name out of the sample, but please see what I added and let me know if it does not work.

File: clusters/cluster_lifecycle/ansible_config_cluster.adoc (hunk 1 above)
Comment (fxiang1) [reply clusters/cluster_lifecycle/ansible_config_cluster.adoc]: @swopebe Good catch. In a regular non-EUS upgrade when we upgrade OCP to the next version the machineconfigpools will upgrade to the next version as well. However, if the user wants to save some time they can optionally pause the machineconfigpools so they don't upgrade to the intermediate release(essentially skipping the intermediate version 4.13.x in this example) and go straight to 4.14.x. Hope that make sense. Let me know if you have more questions. Thanks!

**Review by fxiang1 — APPROVED** [review]

File: clusters/release_notes/whats_new.adoc [hunk 2]
```diff
@@ -36,6 +36,7 @@ Learn about new features and enhancements for Cluster lifecycle with {mce-short}
 
 - Placements do not select managed clusters that you are deleting. To learn more about placements, see xref:../cluster_lifecycle/placement_overview.adoc#placement-overview[Placement overview].
 
+- You can use the `ClusterCurator` resource to perform an easier, automatic upgrades from Extended Update Support (EUS) to EUS. The `spec.upgrade.intermediateUpdate` is added to the `ClusterCurator` resource with the intermediate release value before the final upgrade to the `desiredUpdate`. Find the procedure and the example at xref:../cluster_lifecycle/ansible_config_cluster.adoc#eus-upgrades[Using the _ClusterCurator_ for Extended Update Support (EUS) upgrades].
```
Comment (oafischer) [inline clusters/release_notes/whats_new.adoc]:
```suggestion
- You can use the `ClusterCurator` resource to perform easier, automatic upgrades from Extended Update Support (EUS) to EUS. The `spec.upgrade.intermediateUpdate` is added to the `ClusterCurator` resource with the intermediate release value before the final upgrade to the `desiredUpdate`. Find the procedure and the example at xref:../cluster_lifecycle/ansible_config_cluster.adoc#eus-upgrades[Using the _ClusterCurator_ for Extended Update Support (EUS) upgrades].
```

**Review by oafischer — APPROVED** [review]
One typo, otherwise looks great!

**Pull request merged**